js/Test.js

```diff
@@ -71,6 +71,7 @@ class Test {
         source.loop = true;
         source.loopStart = 0;
         source.loopEnd = 1;
+        source.playbackRate.value = 0.9;
```

Comment:
Looks great, maybe a short comment in the code around why we're doing this? Even though it's clear from the commit message.